ting_subsearch_secondary.module

```diff
@@ -130,18 +146,18 @@ function ting_subsearch_secondary_search($keys, $results) {
         return $message;
       }
 
-      $nonfiction = 1;
+      $nonfiction = (int) 1;
       if (!empty($genre_facets['nonfiktion'])) {
         $nonfiction = $genre_facets['nonfiktion']->getCount();
       }
 
-      $fiction = 0;
+      $fiction = (int) 0;
```

Comment:
Any reason to explicitly cast the type? It's anyway an `int`, unless it's `1.0f`.